### ISSUE TITLE
ADA non compliance

### ISSUE THREAD

**Issue by ashankarhs-ddbang** [uxsolutions/bootstrap-datepicker#2585]
### Expected behaviour
Should behave in the way mentioned in http://pauljadam.com/guides/datepickers.html

### Actual behaviour
This is not been able to be tabbed in and read by screen reader for the blind.

This qualifies border case of **A** compliance on WCAG 2.0. But, need to be **AA** compliant.
